README.md

```diff
@@ -96,6 +96,7 @@ Hope we help each other to complete the [Hacktoberfest](https://hacktoberfest.di
 | 51. | [Formica](https://github.com/felixfaisal/formica)| [Use Docker Volumes instead of Copy](https://github.com/felixfaisal/formica/issues/55) | `Docker` |
 | 52. | [Express-autodocs](https://github.com/prafulla-codes/express-autodocs)| [Adding a sidebar index in standard output](https://github.com/prafulla-codes/express-autodocs/issues/19) | `Javascript ExpressJs` |
 | 53. | [Express-autodocs](https://github.com/prafulla-codes/express-autodocs)| [Beautify standard documentation page](https://github.com/prafulla-codes/express-autodocs/issues/2) | `Javascript ExpressJs`|
+| 54. | [Formica](https://github.com/felixfaisal/formica)| [Create Docker GitHub Action for Dev Environment](https://github.com/felixfaisal/formica/issues/87) | `Docker``GitHub Actions` |
```

Comment:
@felixfaisal Please avoid adding here. It will make `README.md` clumsy. Kindly remove this.

Reply:
@vinitshahdeo Done, On the side note it's better to remove the table of issues from the README entirely as it is misleading